edition = "2023";

package protobuf;

option features.field_presence = EXPLICIT;

message ActorCount {
    string actor_name = 1;
    int32 count = 2;
}

message ActorCounts {
    repeated ActorCount list = 1;
}